src/mqtt_client/mod.rs

```diff
@@ -89,6 +89,31 @@ mod sm {
     }
 }
 
+enum Command<'a> {
+    Query,
+    List,
+    Modify(&'a str),
+}
```

Comment:
It seems like these commands need to be listed via some retained settings so that the client can know what commands are available. This would aid us in the future if we expose new APIs over MQTT.

We could do this via the `alive` topic, or we could just publish the `miniconf` version being used on the `alive` topic.

Any thoughts on this one @jordens?

Reply:
The latter. I'd make `alive` contain the `miniconf` protocol version (thus guaranteeing the availability of the commands) and (probably) also include an application version identifier.

Reply:
But this change to alive can also be done in a separate PR.

Reply:
Let's follow up with this in a separate PR. I'll open an issue.

Reply:
Added in https://github.com/quartiq/miniconf/issues/125